migrator/README.md

```diff
@@ -0,0 +1 @@
+# pay-mta-migrator
```

Comment:
Haha should probably change this to be the hackapp migrator 😂

Reply:
Haha yes... Yes it should